public/app/features/dashboard-scene/settings/variables/components/SelectionOptionsForm.tsx

```diff
@@ -31,6 +35,15 @@ export function SelectionOptionsForm({
         onChange={onMultiChange}
         testId={selectors.pages.Dashboard.Settings.Variables.Edit.General.selectionOptionsMultiSwitch}
       />
+      {onAllowCustomValueChange && ( // backwards compat with old arch, remove on cleanup
```

Comment:
we still have a dependency on this [here](https://github.com/grafana/grafana/blob/main/public/app/features/variables/editor/SelectionOptionsEditor.tsx#L43) so I'm making this optional till we remove the old code, then we can remove this line as well